packages/forms/src/components/Repeater/Repeater.tsx

```diff
@@ -145,95 +178,143 @@ const Fieldset = ({
-                  data-h2-color="base(inherit)"
-                  data-h2-font-weight="base(800)"
-                >
-                  {legend}
-                </p>
-              )
-            }
-            {children}
-          </div>
-          <div
-            data-h2-flex-shrink="base(0)"
+            data-h2-margin="base(x.5 x1 x1 x1)"
             data-h2-display="base(flex)"
             data-h2-flex-direction="base(column)"
-            data-h2-gap="base(x.25, 0)"
-            data-h2-order="base(1)"
+            data-h2-align-items="base(flex-start)"
           >
             <div
               data-h2-display="base(flex)"
-              data-h2-radius="base(s)"
-              data-h2-shadow="base(medium)"
-              data-h2-flex-direction="base(column)"
-              data-h2-align-items="base(center)"
-              data-h2-overflow="base(hidden)"
+              data-h2-justify-content="base(space-between)"
+              data-h2-width="base(100%)"
+              data-h2-margin-bottom="base(x.5)"
             >
-              <ActionButton
-                disabled={disabled || index <= 0}
-                onClick={decrement}
-                decrement
-                animate={!shouldReduceMotion}
-                aria-label={intl.formatMessage(formMessages.repeaterMove, {
-                  from: position,
-                  to: position - 1,
-                })}
+              <div
+                data-h2-display="base(flex)"
+                data-h2-align-items="base(center)"
+                data-h2-margin-left="base(-x.5)"
               >
-                <ChevronUpIcon data-h2-width="base(x1)" />
-              </ActionButton>
-              {!hideIndex && (
-                <span
-                  aria-hidden="true"
-                  data-h2-text-align="base(center)"
-                  data-h2-font-weight="base(700)"
-                  data-h2-margin="base(x.25, 0)"
-                >
-                  {index + 1}
-                </span>
-              )}
-              <ActionButton
-                disabled={disabled || index === total - 1}
-                onClick={increment}
-                animate={!shouldReduceMotion}
-                aria-label={intl.formatMessage(formMessages.repeaterMove, {
-                  from: position,
-                  to: position + 1,
-                })}
+                {!isLocked ? (
+                  <>
+                    <ActionButton
+                      disabled={isFirstItem || previousItemLocked}
+                      onClick={decrement}
+                      decrement
+                      animate={!shouldReduceMotion}
+                      aria-label={intl.formatMessage(
+                        formMessages.repeaterMove,
+                        {
+                          from: position,
+                          to: position - 1,
+                        },
+                      )}
+                      data-h2-font-weight="base(700)"
```

Comment:
I think all the buttons are bold so adding this to the `ActionButton` might be a way to avoid repeating ourselves.

Reply:
Good point 😆 Fixed by [e51e6ab](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8181/commits/e51e6ab11f1085af2d7a5fa68c9f549426c83db3).